experimental/cloudstrapper/playbooks/roles/control/tasks/configure-certs.yaml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: create certs directory
+  file:
+    name: "{{ dirSecretsLocal }}"
+    state: directory
+    mode: '0775'
+
+- name: generate self-signed certs
```

Comment:
What's the workflow for someone who has CA signed certs ? Will they copy it into some directory ?

Reply:
This workflow is currently for self-signed certs. I will plan to add a follow up PR that enables a feature that can use Public CA provided Certs, likely from the same Cloud provider where Orchestrator lands.

Reply:
https://github.com/magma/magma/issues/5110